src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -8,36 +8,32 @@ namespace Stratis.Bitcoin.Configuration.Settings
     /// </summary>
     public class ConnectionManagerSettings
     {
+        /// <summary>Number of seconds to keep misbehaving peers from reconnecting (Default 24-hour ban).</summary>
+        public const int DefaultMisbehavingBantime = 24 * 60 * 60; 
+
         /// <summary>
         /// Default constructor.
         /// </summary>
         public ConnectionManagerSettings()
         {
+            this.Connect = new List<IPEndPoint>();
+            this.AddNode = new List<IPEndPoint>();
+            this.Listen = new List<NodeServerEndpoint>();
         }
 
         /// <summary>List of exclusive end points that the node should be connected to.</summary>
-        public List<IPEndPoint> Connect
-        {
-            get; set;
-        } = new List<IPEndPoint>();
+        public List<IPEndPoint> Connect { get; set; }
```

Comment:
good, can we agree on moving these above the constructor
some kind of order like
- constants 
- static fields
- fields
- auto-properties
- properties
- abstract methods
- static constructor
- constructor
- methods

Reply:
Hmm this are public properties, normally above the constructor you have only private and const no?

Reply:
i don't know what is "normally", we don't have a policy for that :-) so i'm just sharing my personal preference (and also i'm building and refactoring my code as such)

if you want to do it differently, but don't want to have a policy for that, then ignore me

if you want to have a policy for that, let's create issue to discuss that